sormas-backend/src/main/resources/utils/deleteCases.sql

```diff
@@ -0,0 +1,156 @@
+DROP TABLE IF EXISTS delete_cleanup;
```

Comment:
Add `BEGIN TRANSACTION;` instead of the `DROP TABLE` statement to encapsulate the whole delete run in one transaction.

Finish the whole script with `COMMIT; ` and an empty line after that.